tools/proto-compiler/src/main.rs

```diff
@@ -80,7 +80,15 @@ fn main() {
         "super::super::google::protobuf::Timestamp",
     );
     println!("[info] => Creating structs.");
-    pb.compile_protos(&protos, &proto_includes_paths).unwrap();
+    tonic_build::configure()
+        .out_dir(&out_dir)
+        .build_client(false)
+        .build_server(true)
```

Comment:
What if we want to enable client-side gRPC bindings later, and feature-gate client and server sides independently? I think it's possible to introduce "grpc-server" and "grpc-client" features later, with "grpc" dependent on both, so this would be backward compatible.